src/styles/index.ts

```diff
@@ -2794,6 +2794,39 @@ const styles = (theme: ThemeColors) =>
             fontWeight: FontUtils.fontWeight.bold,
         },
 
+        subscriptionSettingsSectionTitle: {
```

Comment:
Can we try to dry this up? I feel like we already have existing style utility classes we can use, or in the very least, let's not make this specific to "subscriptionSettings" in case we reuse this pattern elsewhere.

Reply:
Good point, I noticed it's exactly the same as `textStrong` so I used this instead